### DISCUSSION TITLE
GET datasets changed after a given date

### DISCUSSION THREAD

**Discussion by swhume** [cdisc-org/DataExchange-DatasetJson-API#5] [User Stories]
As a sponsor, I want to get a listing of all datasets that have changed as of a certain date so that I can retrieve datasets that have updates

**Comment by JozefAerts** [comment]
datasets changed or records changed?

**Reply by swhume** [reply]
records within the dataset have been changed/added/removed

**Comment by mhungria** [comment]
Could it be both, changes to records or the dataset itself, for this endpoint?

**Comment by lexjansen** [comment]
How about changes to the metadata? Like a variable added or removed.

**Reply by JozefAerts** [reply]
Good catch! What we often see is that during the SDTM development stage, additional variables are added, or sometimes, removed.

**Reply by mhungria** [reply]
I think that may be part of separate endpoints that require a more complex response. I'm referring to Sam's summary on Slack: (1) basic dataset exchange only, and (2) the ability to query datasets to retrieve useful information. ... The first 1 is relatively simple and focuses exclusively on data exchange.
For instance, #16 would be part of the second category.

**Reply by lexjansen** [reply]
I guess when a variable is added or removed, every record will have changed, so that still fits with (1).
